src/routes/Item.svelte

```diff
@@ -95,24 +133,21 @@
 		} else if (event.key === 'Enter') {
 			isExpanded = !isExpanded;
 		} else if (event.key === 'ArrowUp') {
-			dispatch('up');
+			onSelected(-1);
 		} else if (event.key === 'ArrowDown') {
-			dispatch('down');
+			onSelected(1);
 		} else if (event.key === 'ArrowRight') {
-			item.quantity++;
+			onQuantityChange(item.quantity + 1);
```

Comment:
nothing to fix in this pr, but just a note that ive started thinking about how to handle this in the future, where you might have multiple of the same item with different dates added & u want to change the quantity

i have a general idea for the data structure  (1 layer of nesting, grouping together quantities of items added on the same day) and how increment / decrement might work by default (if decreasing, subtract from the quantity of the oldest group first, and once that reaches 0, subtract from the next oldest and so on; and if increasing, adding to the quantity of a group whose date matches the current date, creating a new group if none exists)